alembic/versions/ed436e5e9cd6_autho2.py

```diff
@@ -0,0 +1,28 @@
+"""autho2
```

Comment:
All the new migration files look incredibly sus
From my understanding, user table should be created in an upgrade and those foreign keys added to necessary tables
Then downgrade will consist of deleting the foreign keys and the table

Reply:
I've not been able to successfully migrate this, we could possibly check this out together later... having issues with FK and i haven't worked on it since the last time we spoke.

Reply:
Alright no problem